manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -6,9 +6,15 @@ rules:
 - apiGroups: [""]
   resources: ["nodes"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["config.openshift.io"]
+  resources: ["clusterversions", "featuregates"]
+  verbs: ["*"]
 - apiGroups: ["machineconfiguration.openshift.io"]
   resources: ["machineconfigs", "controllerconfigs"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["machineconfiguration.openshift.io"]
+  resources: ["machineconfignodes", "machineconfignodes/status"]
```

Comment:
curious, isn't just saying machineconfignodes resource will also include machineconfignodes/status sub-resource or this is created as a separate resource?

Reply:
If you do .`UpdateStatus` I think you need to explicitly say we have access to that sub-resource. A couple other objects in the MCO use the same verbiage here.